docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │   601     30  female      2  own      little           moderate         ⋯
+   2 │   602     34  female      1  free     little           moderate
+   3 │   603     28  female      3  own      little           NA
+   4 │   604     23  female      2  own      little           rich
+   5 │   605     22  male        2  own      quite rich       little           ⋯
+   6 │   606     74  male        3  own      little           NA
+   7 │   607     50  female      2  free     moderate         moderate
+   8 │   608     33  male        2  own      little           NA
+  ⋮  │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮         ⋱
+ 393 │   993     30  male        3  own      little           little           ⋯
+ 394 │   994     50  male        2  own      NA               NA
+ 395 │   995     31  female      1  own      little           NA
+ 396 │   996     40  male        3  own      little           little
+ 397 │   997     38  male        2  own      little           NA               ⋯
+ 398 │   998     23  male        2  free     little           little
+ 399 │   999     27  male        2  own      moderate         moderate
+                                                  3 columns and 384 rows omitted
+```
+
+If you need to match a specific subset of values, then the `in` function can be applied:
+```jldoctest dataframe
+julia> german[in.(german.id, Ref(Set([1, 6, 908, 955]))), :]
+5×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     1      1  male        2  own      NA               little           ⋯
+   2 │     1      1  female      2  own      little           moderate
+   3 │     6     53  male        2  own      quite rich       NA
+   4 │   908     46  female      1  own      little           NA
+   5 │   955     57  female      3  rent     rich             little           ⋯
+                                                               3 columns omitted
+```
+
+Equivalently, the `in` function can be called with a single argument to create a function object that
+tests whether each value belongs to the subset (partial application of `in`):
+`german[in(Set([1, 6, 908, 955])).(german.id), :]`
+
+## Not, Between, Cols, findall and All selectors
+
+Finally, you can use `Not`, `Between`, `Cols`, and `All` selectors in more complex column selection
+scenarioes (note that `Cols()` selects no columns while `All()` selects all columns). Now we will 
+see how `Cols` and `Between` can be used to select columns of a data frame. `All()` allows us to select  
+all columns of `DataFrame` while `Between` selector allow us to specify a range of columns (we can 
```

Comment:
lack of logic linking in these two sentences.

Reply:
you keep mixing `All` and `Cols` usage. It would be best if you discussed all selectors in a sequence without mixing the explanations.